packages/amplify_flutter/lib/categories/amplify_api_category.dart

```diff
@@ -45,6 +45,20 @@ class APICategory {
         : throw _pluginNotAddedException("Api");
   }
 
+  GraphQLSubscriptionOperation<T> subscribe<T>(
+      {@required GraphQLRequest<T> request,
+      @required Function(GraphQLResponse<T>) onData,
+      Function() onEstablished,
+      Function(dynamic) onError,
+      Function() onDone}) {
+    return plugins[0].subscribe(
+        request: request,
+        onEstablished: onEstablished,
+        onData: onData,
+        onError: onError,
+        onDone: onDone);
+  }
```

Comment:
To be consistent with other methods in this file, throw an exception if number of plugins != 1

Reply:
Done